tests/Kernel/fixtures/rendering.yml

```diff
@@ -642,15 +643,14 @@
       length: 6
       meta:
         - "Meta 1"
-        - "Meta 2"
   assertions:
     count:
       'article.ecl-content-item div.ecl-content-item__title a.ecl-link[href="http://example.com"]': 1
       'article.ecl-content-item div[role="img"].ecl-u-d-lg-block': 0
       'article.ecl-content-item div.ecl-content-item__image__before': 0
       'article.ecl-content-item div.ecl-content-item__image__after': 0
```

Comment:
Also let's add a test that proves that, when no field list field is passed, then we don't print any field list.

Reply:
I added a test checking 
      'article > div .ecl-description-list .ecl-description-list__term': 0
      'article > div .ecl-description-list .ecl-description-list__definition': 0